relayer/processor/metrics.go

```diff
@@ -104,5 +115,13 @@ func NewPrometheusMetrics() *PrometheusMetrics {
 			Name: "cosmos_relayer_client_trusting_period_seconds",
 			Help: "The trusting period (in seconds) of the client",
 		}, clientTrustingPeriodLables),
+		UnrelayedPackets: registerer.NewGaugeVec(prometheus.GaugeOpts{
+			Name: "cosmos_relayer_unrelayed_packets",
+			Help: "Current number of unrelayed packets on both the source and destinations chains on a specific path and channel",
+		}, unrelayedSeqsLabels),
+		UnrelayedAcks: registerer.NewGaugeVec(prometheus.GaugeOpts{
+			Name: "cosmos_relayer_unrelayed_acks",
+			Help: "Current number of unrelayed acknoledgments on both the source and destinations chains on a specific path and channel",
```

Comment:
In, "Current number of unrelayed acknoledgments on both the source and destinations chains on a specific path and channel", `acknoledgments` should be `acknowledgements`.

Also, the phrasing seems a bit off. Something like, "Current number of unrelayed acknowledgements on both the source and destination chains for a specific path and channel" seems more correct